SimperiumTests/SPCoreDataStorageTests.m

```diff
@@ -111,21 +111,25 @@ - (void)testInsertingChildEntitiesWhileDeletingRootEntity
     
 	dispatch_async(commentBucket.processorQueue, ^{
 		id<SPStorageProvider> threadSafeStorage = [self.storage threadSafeStorage];
-		[threadSafeStorage performSafeBlockAndWait:^{
-            
-            for (NSString* simperiumKey in postKeys) {
-                
+
+        for (NSString* simperiumKey in postKeys) {
```

Comment:
You changed behavior here to have a save execute after every key instead of after all the keys were processed. Is that expected? I know this is a unit test but I figured I'd check. 😄

Reply:
On purpose! The purpose of this test was to try to cause a deadlock. Previous implementation was _less_ likely to trigger bad scenarios.

Thanks for the heads up!!